pkg/operator/config_reloader.go

```diff
@@ -221,6 +223,34 @@ func CreateConfigReloader(name string, options ...ReloaderOption) v1.Container {
 		resources.Limits[v1.ResourceMemory] = resource.MustParse(configReloader.config.MemoryLimit)
 	}
 
+	livenessProbeHandler := v1.ProbeHandler{
+		HTTPGet: &v1.HTTPGetAction{
+			Path: path.Clean("/healthz"),
+			Port: intstr.FromInt(configReloaderPort),
+		},
+	}
+
+	readinessProbeHandler := v1.ProbeHandler{
+		HTTPGet: &v1.HTTPGetAction{
+			Path: path.Clean("/ready"),
+			Port: intstr.FromInt(configReloaderPort),
+		},
+	}
+
+	livenessProbe := &v1.Probe{
+		ProbeHandler:     livenessProbeHandler,
+		TimeoutSeconds:   3,
+		FailureThreshold: 10,
+	}
+
+	readinessProbe := &v1.Probe{
+		ProbeHandler:        readinessProbeHandler,
+		InitialDelaySeconds: 3,
+		TimeoutSeconds:      3,
+		PeriodSeconds:       5,
+		FailureThreshold:    10,
+	}
+
```

Comment:
could we move this to a different method say - `addProbes(c v1.Container) v1.Container` so that the call would be 
```
c := v1.Container {
   ...
}

if !config.runOnce && config.EnableProbes {
   return addProbes(c)
}
 return c
```

Reply:
add `addProbes (c *v1.Container) *v1.Container` method